src/lib/openapi/spec/personal-dashboard-schema.ts

```diff
@@ -0,0 +1,51 @@
+import type { FromSchema } from 'json-schema-to-ts';
+
+export const personalDashboardSchema = {
```

Comment:
early iteration of the schema. We'll keep improving it over time (especially the projects part)